From 9ee4dce5f349ee63bdd9b9e2500b337c8a830905 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Tue, 21 Mar 2023 20:31:10 -0300
Subject: [PATCH 1/4] GENIO: mt8195: board: define dfu_alt_info for capsule
 update support

Files are known and by default it is expected to boot from eMMC.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 board/mediatek/mt8195/mt8195_demo.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board/mediatek/mt8195/mt8195_demo.c b/board/mediatek/mt8195/mt8195_demo.c
index 066433de404..2fb0558e3b5 100644
--- a/board/mediatek/mt8195/mt8195_demo.c
+++ b/board/mediatek/mt8195/mt8195_demo.c
@@ -17,6 +17,8 @@
 static struct efi_fw_image fw_images[MT8195_UPDATABLE_IMAGES] = {0};
 
 struct efi_capsule_update_info update_info = {
+	.dfu_string = "mmc 0=bl2.img raw 0x0 0x100 mmcpart 1;"
+			"fip.bin part 0 1",
 	.images = fw_images,
 };
 
-- 
2.39.1

